polars/polars-row/src/row.rs

```diff
@@ -1,18 +1,23 @@
-use crate::sort_field::SortField;
+use arrow::array::BinaryArray;
+use arrow::datatypes::DataType;
+use arrow::offset::Offsets;
+
+#[derive(Clone)]
+pub struct SortField {
+    /// Whether to sort in descending order
+    pub descending: bool,
+    /// Whether to sort nulls first
+    pub nulls_last: bool,
+}
```

Comment:
@ritchie46 This (removing `data_type`) seems to break the tests in encode.rs https://github.com/pola-rs/polars/blob/master/polars/polars-row/src/encode.rs#L203? Weird the CI doesn't catch it (you can simply run `cargo test` with no arguments to verify).

Reply:
Thanks for the heads up. Will fix it.